fspiop-api/documents/API Definition v2.0-alpha.md

```diff
@@ -3209,485 +3630,550 @@ This section defines elements types used by the API.
 
-#### 7.3.35 TransferState
+#### 7.3.36 TransferState
 
-[Table 80](#table-80) contains the data model for the element **TransferState**.
+[Table 93](#table-93) contains the data model for the element **TransferState**.
 
-###### Table 80
+###### Table 93
 
 | **Name** | **Cardinality** | **Format** | **Description** |
 | --- | --- | --- | --- |
-| **TransferState** | 1 | Enum of String(1..32) | See [Section 7.5.13](#7513-transferstate) (TransferState) for more information on allowed values. |
+| **TransferState** | 1 | Enum of String(1..32) | See [Section 7.5.14](#7514-transferstate) (TransferState) for more information on allowed values. |
 
-**Table 80 -- Element TransferState**
+**Table 93 -- Element TransferState**
 
 ### 7.4 Complex Types
 
 This section describes complex types used by the API.
 
 #### 7.4.1 AuthenticationInfo
 
-[Table 81](#table-81) contains the data model for the complex type **AuthenticationInfo**.
+[Table 94](#table-94) contains the data model for the complex type **AuthenticationInfo**.
 
-###### Table 81
+###### Table 94
 
 | **Name** | **Cardinality** | **Format** | **Description** |
 | --- | --- | --- | --- |
 | **authentication** | 1 | AuthenticationType | Type of authentication. | 
 | **authenticationValue** | 1 | AuthenticationValue | Authentication value. | 
 
-**Table 81 -- Complex type AuthenticationInfo**
+**Table 94 -- Complex type AuthenticationInfo**
+
+#### 7.4.2 Charge
+
+[Table 95](#table-95) contains the data model for the complex type **Charge**.
+
+###### Table 95
+
+| **Name** | **Cardinality** | **Format** | **Description** |
+| --- | --- | --- | --- |
+| **chargeType** | 1 | String(1..32) | A descripition of the charge that is being levied |
+| **sourceAmount** | 1 | Amount | The amount of the charge which is being levied, expressed in the source currency. |
+| **targetAmount** | 1 | Amount | The amount of the charge which is being levied, expressed in the target currency. |
+
+**Table 95 -- Complex type Charge**
+
+#### 7.4.3 Conversion
+
+[Table 95](#table-95) contains the data model for the complex type **Conversion**.
+
+###### Table 95
+
+| **Name** | **Cardinality** | **Format** | **Description** |
+| --- | --- | --- | --- |
+| **conversionId** | 1 | CorrelationId | An end-to-end identifier for the conversion request. |
+| **determiningTransferId** | 0..1 | CorrelationId | The transaction ID of the transfer on whose success this currency conversion depends. If this is a bulk currency conversion which is not dependent on a transfer, then this field should be omitted. |
+| **counterPartyFsp** | 1 | FspId | The ID of the FXP performing the conversion. |
+| **amountType** | 1 | AmountType | This is the AmountType for the base transaction, as described in Section 7.3.1 above.<ln>
+If it is set to SEND, then any charges levied by the FXP as part of the transaction will be deducted by the FXP from the amount shown for the target party in the conversion.<ln>
```

Comment:
I don't think this should be on it's own line.. See how it is done for IndividualQuote.